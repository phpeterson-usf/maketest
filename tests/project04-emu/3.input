-f fib_iter -a 20
