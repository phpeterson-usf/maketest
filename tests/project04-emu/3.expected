6765 6765 6765
